.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+on:
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+
+        env:
+            ZOTONIC_DIR: /tmp/zotonic
+
+        services:
+            postgres:
+                image: postgres:9.5.5-alpine
+                env:
+                    POSTGRES_USER: zotonic
+                ports:
+                    - 5432:5432
+                options: >-
+                    --health-cmd pg_isready
+                    --health-interval 10s
+                    --health-timeout 5s
+                    --health-retries 5
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2.4.0
+
+            -   name: Set up Erlang
+                uses: erlef/setup-beam@v1.10.0
+                with:
+                    otp-version: "21.3"
+
+            -   name: Install and configure Zotonic
+                run: |
+                    git clone https://github.com/zotonic/zotonic.git $ZOTONIC_DIR --branch 0.x
+                    mkdir -p ~/.zotonic/0 ${ZOTONIC_DIR}/user
+                    sed "s|\.\.\/ginger/modules|$GITHUB_WORKSPACE/modules|" config/zotonic.config > ~/.zotonic/0/zotonic.config
+
+            -   name: Install dependencies
+                run: |
+                    git clone https://github.com/driebit/mod_elasticsearch.git modules/mod_elasticsearch
+
+            -   name: Compile
+                working-directory: ${{ env.ZOTONIC_DIR }}
+                run: make all
+
+            -   name: Run tests
+                working-directory: ${{ env.ZOTONIC_DIR }}
+                env:
+                    ERL: "erl -noshell"
+                run: $GITHUB_WORKSPACE/scripts/runtests.sh
```

Comment:
I didn't work with github workflows yet, so this is a nice introduction. Clear script!

Reply:
The Slack notification was removed, is that intentional/checked with Marco? Maybe it is indeed not needed anymore - though I am used to see them signalling successful tests..are these the ones sent to the room ginger-notifications?

Reply:
@robvandenbogaard The Slack notification is only saying whether the tests failed/succeeded, so it’s really a non-essential addition to the feedback from GitHub itself. Because of this, I’d suggest to forget about the notification for now.

If you want to [re-add the notification](https://axolo.co/blog/p/top-4-github-action-slack-integration), that would require some configuration on the Driebit Slack (which I have no admin access to). So first let’s see if it’s really needed, and only if so, set up the Driebit Slack.

Reply:
I now see that we probably did not have these notifications for some time, so it is ok to remove it (was confusing the notifications about test/acceptance getting updated).